consoles/sshVirtsh.pm

```diff
@@ -384,7 +384,7 @@ sub add_disk {
                       "echo File $file_basename is being copied by other process, sleeping for 60 seconds; sleep 60;" .
                       'done;' .
                       'else ' .
-                      "cp /vmfs/volumes/$vmware_nfs_datastore/$nfs_dir/$file_basename $vmware_openqa_datastore;" .
+                      "rsync --checksum -av /vmfs/volumes/$vmware_nfs_datastore/$nfs_dir/$file_basename $vmware_openqa_datastore;" .
```

Comment:
There's a misunderstanding. There's no `rsync` on ESXi and it's not an trivial enterprise to get it there.

Reply:
Ahh It sounded too easy to be true :)

Reply:
I had a hope =)